webpay/auth/views.py

```diff
@@ -39,6 +40,15 @@ def reset_user(request):
         # This isn't strictly necessary since permissions are reset on
         # login but it's good for paranoia.
         del request.session['mkt_permissions']
+
+    # Set the starting timestamp of the reset for later use when verifying
+    # their Firefox Accounts password entry.
+    request.session.setdefault('user_reset', {})
```

Comment:
hmm, I was thinking that you could do things out of order but that doesn't make sense. You have to begin a reset before you verify it. I'll fix.